apps/voting/app/src/vote-description-utils.js

```diff
@@ -0,0 +1,135 @@
+import React, { useMemo } from 'react'
+import styled from 'styled-components'
+import { GU, Link, useTheme } from '@aragon/ui'
+import LocalIdentityBadge from './components/LocalIdentityBadge/LocalIdentityBadge'
+
+export function renderDescription(path) {
+  const theme = useTheme()
+  const description = useMemo(() => {
+    return path
+      ? path.map((step, index) =>
+          renderForwardingStep(step, [index + 1], theme)
+        )
+      : ''
+  }, [path, theme])
+  return <React.Fragment>{description}</React.Fragment>
+}
+
+function renderForwardingStep(step, depth, theme) {
+  const app =
```

Comment:
Perhaps we can leave a TODO note with the app variable to eventually move to using an AppBadge for this component.